### DISCUSSION TITLE
3D Printed accessories

### DISCUSSION THREAD

**Discussion by moarkaboom** [CrashOverride85/zc95#122] [Show and tell]
After finishing my first unit, I found I  would like some parts to help me with the assembly. I designed (big word for what it is) two parts that may help you with construction:

**10 mm Riser**
[10mm_Riser.zip](https://github.com/user-attachments/files/17243636/10mm_Riser.zip)
This is quite similar to the LED risers which I highly recommend you also print. It helps set the distance of the front panel when attaching it to the control panel.
![WhatsApp Image 2024-10-02 at 17 55 46](https://github.com/user-attachments/assets/1b0af994-3be1-4fd1-b74b-9381804b4958)

**Button Washer**
[ButtonWasher.zip](https://github.com/user-attachments/files/17243639/ButtonWasher.zip)
The LED Buttons dont really fit that well and there is some play. I solved this by clipping on this washer behind it to make everything a bit more snug. 
![1b20c7b4-d7b0-45e1-a308-5cd5ce8b6259](https://github.com/user-attachments/assets/00760655-37d8-4d9a-abe8-8fb3a433dbff)

**Comment by electro991** [comment]
@moarkaboom thank for the 👍 on the led risers. 
i have built one more unit here and used your spacers and button washers. They worked awesome. Thanks you. It would be great if you came up with a retrofit option for the button washers so one could install those on a unit that has the button already wired. I would definitely use.

**Reply by electro991** [reply]
@moarkaboom I edited your stl file for the button washer "retrofit". Makes existing builds buttons 99% more steady in their sockets. There's a tiny notch that allows wires to be inserted into the holes one by one. and then slowly push in. Works great. To insert the one near the screen, I had to just loosen the screws a little on the screen and it slides in no problem. Thanks for this great idea!
[retrofit button washer v1.zip](https://github.com/user-attachments/files/18417719/retrofit.button.washer.v1.zip)
